src/main/ipc/messages.ts

```diff
@@ -4,7 +4,8 @@ enum IPCMessages {
   GET_KEYSTORE = 'GET_KEYSTORE',
   KEYSTORE_EXIST = 'KEYSTORE_EXIST',
   REMOVE_KEYSTORE = 'REMOVE_KEYSTORE',
-  GET_BTC_ADDRESS = 'GET_BTC_ADDRESS'
+  GET_BTC_ADDRESS = 'GET_BTC_ADDRESS',
+  GET_BNB_ADDRESS = 'GET_BNB_ADDRESS'
```

Comment:
To avoid x `GET_XYZ_ADDRESS` summarize it to one `GET_LEDGER_ADDRESS` and pass `chain: Chain` as parameter (besides `network`)